src/docs-md/basics/decorators.md

```diff
@@ -80,19 +80,35 @@ in JSX you set an attribute using camelCase:
 
 They can also be accessed via JS from the element.
 
-```typescript 
+```typescript
 const todoListElement = document.querySelector('todo-list');
 console.log(todoListElement.myHttpService); // MyHttpService
 console.log(todoListElement.color); // blue
 ```
 
-It's important to know, that `@Prop` is immutable from inside the component logic. Once a value is set by a user, the component cannot update it internally.
+### Prop value mutability
+
+It's important to know, that `@Prop` is _by default_ immutable from inside the component logic. Once a value is set by a user, the component cannot update it internally.
+
+However, it's possible to explicitly let a `@Prop` allow its value to be modified from inside the component, by declaring it as **mutable**, as in example blow:
```

Comment:
`However, it's possible to explicitly let` should be `However, it's possible to explicitly allow`

Reply:
done. I rephrased as `However, it's possible to explicitly allow a @Prop to be mutated...`